code/game/gamemodes/objective.dm

```diff
@@ -1026,4 +1026,50 @@ datum/objective/heist/inviolate_death
 	explanation_text = "Follow the Inviolate. Minimise death and loss of resources."
 	check_completion()
 		if(vox_kills > MAX_VOX_KILLS) return 0
-		return 1
\ No newline at end of file
+		return 1
+
+// Traders
+
+/datum/objective/trade // Yes, I know there's no check_completion. The objectives exist only to tell the traders what to get.
+	proc/choose_target(var/station)

```

Comment:
i must express my discontent with adding more relative pathing to this file despite following the established styling